<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="RAMESHRIYO" ignored="1" name="Default test(failed)" tests="2" failures="1" timestamp="31 Dec 2022 12:13:53 GMT" time="0.061" errors="0">
  <testcase name="tc1" time="0.007" classname="org.group.Employee"/>
  <testcase name="tc2" time="0.006" classname="org.group.Employee">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:87)
at org.junit.Assert.assertTrue(Assert.java:42)
at org.junit.Assert.assertTrue(Assert.java:53)
at org.group.Employee.tc2(Employee.java:19)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- tc2 -->
  <testcase name="tc0" time="0.0" classname="org.group.Employee">
    <ignored/>
  </testcase> <!-- tc0 -->
</testsuite> <!-- Default test(failed) -->
